tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1491,7 +1491,7 @@ def evaluate_predictions(self, y_true, y_pred, sample_weight=None, silent=False,
         return self._learner.evaluate_predictions(y_true=y_true, y_pred=y_pred, sample_weight=sample_weight, silent=silent,
                                                   auxiliary_metrics=auxiliary_metrics, detailed_report=detailed_report)
 
-    def leaderboard(self, data=None, extra_info=False, extra_metrics=None, only_pareto_frontier=False, skip_score=False, silent=False):
+    def leaderboard(self, data=None, extra_info=False, extra_metrics=None, only_pareto_frontier=False, skip_score=False, silent=False) -> pd.DataFrame:
```

Comment:
nit: add parameter types too

Reply:
added, note type hint for data will wait for future PR due to multiple types support